src/pudl_rmi/connect_ferc1_to_eia.py

```diff
@@ -1196,3 +1198,106 @@ def check_match_consistentcy(connects_ferc1_eia, train_df, match_type='all'):
             f"{consistency_one_cap_ferc:.1%} to {consist_one_cap_ferc:.1%}"
         )
     return count
+
+
+def calc_annual_capital_addts_ferc1(steam_df, window=3):
+    """
+    Calculate annual capital additions for FERC1 steam records.
+
+    Convert the capex_total column into annual capital additons the
+    `capex_total` column is the cumulative capital poured into the plant over
+    time. This function takes the annual difference should generate the annual
+    capial additions. It also want generates a rolling average, to smooth out
+    the big annual fluxuations.
+
+    Args:
+        steam_df (pandas.DataFrame): result of `prep_plants_ferc()`
+
+    Returns:
+        pandas.DataFrame: augemented version of steam_df with two additional
+        columns: `capex_annual_addt` and `capex_annual_addt_rolling`.
```

Comment:
What do `addt` and `addts` mean?

Reply:
addition(s)! i'll define this somewhere. unless you think I should spell this out entirely

Reply:
I think just spelling it out is probably best. Failing that it looks like "add" is probably the most common abbreviation for "addition"